analyze_heap.py

```diff
@@ -180,6 +180,14 @@ def _graph(args: argparse.Namespace) -> None:
         censor_list: list[str] | None = None
         if args.censor:
             censor_list = args.censor.split(",")
+
+        def _censor(string: str) -> str:
```

Comment:
fyi this needs to have both the typename and the payload, because the censoring check occurs on the typename.

also fyi when in censor mode we didn't send any payloads (did we?)

Reply:
Yeah, I am extending it to censor both typename and the payload, now the the payload can contain stacktraces, but also string prompts which we might want to censor. Just calling this function once for each.

Reply:
oh my point was that if we are in censor mode, we never include the payload (stacktraces, prompts, etc are all elided)

Reply:
Got it